méli-mélo/th-choose-canada/choose-canada.html

```diff
@@ -0,0 +1,91 @@
+---
+title: Choose Canada theme
+dateModified: 2025-03-26
+description: Choose Canada CSS styles
+lang: en
+css:
+- style.css
+layout: no-container
+---
+
+<div class="bg-cover bg-center choose-canada-banner" data-bgimg-srcset="images/banner.jpg 767w">
```

Comment:
Do we still need the blank background, as per the source code sample in the index.html